lib/tasks/deploy.rake

```diff
@@ -134,12 +134,19 @@ namespace :deploy do
   end
 
   task :rolling_deploy do
```

Comment:
Would you rather we have a `validate_rolling_deploy_options` dependent task? I don't believe we perform any validation on the values of other rolling deploy options, could take a swing at that. Or just put the one validation at the top of this task if you think it's better to keep this changeset smaller.